internal/commands/edit.go

```diff
@@ -0,0 +1,325 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"hellper/internal/app"
+	"hellper/internal/config"
+	"hellper/internal/model"
+	"strconv"
+	"strings"
+	"time"
+
+	"hellper/internal/bot"
+	"hellper/internal/log"
+
+	"github.com/slack-go/slack"
+)
+
+// OpenEditIncidentDialog opens a dialog on Slack, so the user can edit an incident
+func OpenEditIncidentDialog(ctx context.Context, app *app.App, channelID string, triggerID string) error {
+	var (
+		dateLayout = "2006-01-02T15:04:05-0700"
+		initValue  = ""
```

Comment:
Could you rename it to something that indicates it's the start date?

Something like `startDateString`